apps/gp2-server/test/data-providers/working-group.data-provider.test.ts

```diff
@@ -77,5 +78,160 @@ describe('Working Group Data Provider', () => {
         leadingMembers: '',
       });
     });
+
+    describe('Parsing', () => {
+      test('the working group is parsed', () => {
+        const workingGroup = getGraphQLWorkingGroup();
+        const workingGroupDataObject =
+          parseWorkingGroupToDataObject(workingGroup);
+        const expected = getWorkingGroupDataObject();
+        expect(workingGroupDataObject).toEqual(expected);
+      });
+
+      test('the members in working group are parsed', () => {
+        const workingGroup = getGraphQLWorkingGroup();
+        const workingGroupWithMembers = {
+          ...workingGroup,
+          flatData: {
+            ...workingGroup.flatData,
+            members: [
+              {
+                id: 'member-id',
+                role: 'Chair',
+                user: [
+                  {
+                    id: '42',
+                    created: '2021-01-01T00:00:00Z',
+                    lastModified: '2021-01-01T00:00:00Z',
+                    version: 1,
+                    flatData: {
+                      firstName: 'Tony',
+                      lastName: 'Stark',
+                      avatar: null,
+                    },
+                  },
+                ],
+              },
+            ],
+          },
+        };
+        const workingGroupDataObject = parseWorkingGroupToDataObject(
+          workingGroupWithMembers,
+        );
+        const expected = getWorkingGroupDataObject({
```

Comment:
you don't really need to compare the entire result, you're just interested in members here, right? well in that case you could either extract the members for the expectation or use .toMatchObject and use a partial comparison

Reply:
done